crates/python/src/api.rs

```diff
@@ -221,3 +234,16 @@ pub fn get_quilc_version(py: Python<'_>) -> PyResult<&PyAny> {
         Ok(version)
     })
 }
+
+#[pyfunction]
+pub fn list_quantum_processors(py: Python<'_>) -> PyResult<&PyAny> {
```

Comment:
Why is this name different than the Rust function, if they do the same thing?

Also, these are IDs rather than names.